### ISSUE TITLE
Description of terms within file format

### ISSUE THREAD

**Issue by skinkie** [alicevision/meshroom-manual#12]
As developer/user I am currently reviewing the .sfm fileformat. I want to know what for example an intrinsicId is. OpenMVG is obviously a good place to start https://readthedocs.org/projects/openmvg/

**Comment by natowi** [comment]
Yes, I have some notes on the sfm format.

intrinsicId is used for images that share the same camera intrinsics. 

https://github.com/alicevision/meshroom/issues/683

**Comment by skinkie** [comment]
I was very surprised to see 2 intriniscs from the same smartphone camera. But I think this is the difference between focus macro vs infinity.

**Comment by natowi** [comment]
Yes. We can not use the same camera intrinsics for a different focus

**Comment by simogasp** [comment]
More generally, the sfm file is a mapping  of the SfmData structure into a son format, normally the same name for the fields is preserved
https://github.com/alicevision/AliceVision/blob/develop/src/aliceVision/sfmData/SfMData.hpp
https://github.com/alicevision/AliceVision/blob/develop/src/aliceVision/sfmData/View.hpp
